lib/src/Pos/Launcher/Resource.hs

```diff
@@ -293,6 +295,12 @@ allocateNodeContext ancd txpSettings = do
     let slotDuration :: Integer
         slotDuration = toMicroseconds . bvdSlotDuration $ gdBlockVersionData genesisData
     ncSubscriptionKeepAliveTimer <- newTimer $ 3 * fromIntegral slotDuration
+
+    _mmRollbacks <- liftIO $ Metrics.createGauge "cardano.BlockRollbacks" ekgStore
+    _mmSscFailures <- liftIO $ Metrics.createCounter "cardano.SCCComputationFailures" ekgStore
+    _mmIgnoredCommitments <- liftIO $ Metrics.createGauge "cardano.IgnoredCommitments" ekgStore
+    let mm = MisbehaviorMetrics {..}
```

Comment:
I'd define `MonadIO m => EKG.Store -> m MisbehaviorMetrics` in `Pos.Reporting.MemState` (near `MisbehaviorMetrics` definition).
Also please use `Pos.System.Metrics.Constants.withCardanoNamespace` for metric names.